tools/publish_image_to_docker_hub.sh

```diff
@@ -51,14 +51,38 @@ echo "release_tag: $RELEASE_TAG";
 
 docker pull $IMAGE;
 docker tag $IMAGE openvino/model_server:$RELEASE_TAG;
+docker run openvino/model_server:$RELEASE_TAG --version;
+echo "Press y to continue..."
```

Comment:
I would check the release_tag against current date: year == year, minor version >= current month/ 4 and print warning.